Dockerfile

```diff
@@ -10,13 +10,14 @@ LABEL description "HPFeeds CIFv3 handler is a tool for generating CIFv3 entries
 LABEL authoritative-source-url "https://github.com/CommunityHoneyNetwork/hpfeeds-cif"
 LABEL changelog-url "https://github.com/CommunityHoneyNetwork/hpfeeds-cif/commits/master"
 
-ENV playbook "hpfeeds-cif.yml"
 
-RUN apt-get update \
-       && apt-get install -y ansible
+COPY requirements.txt /opt/requirements.txt
+
+RUN apt-get update && apt-get install -y gcc git mongodb python3-dev python3-pip runit
+RUN pip3 install -r /opt/requirements.txt
+RUN pip3 install git+https://github.com/CommunityHoneyNetwork/hpfeeds3.git
```

Comment:
@amerck Should this reference a specific commit/tag so we can have consistent builds?

Reply:
It should, once we have a tagged release.